i18n/ru.json

```diff
@@ -1,7 +1,8 @@
 {
 	"@metadata": {
 		"authors": [
-			"Mini Kostik"
+			"Mini Kostik",
+			"Naleksuh"
```

Comment:
Was Naleksuh involved in this translation?

Reply:
Yes.
Those translations come from Transfer from the InternationalScratchWiki project where he is a translator.